app/api/public-cloud/aws-roles/deleteUser/route.ts

```diff
@@ -0,0 +1,34 @@
+import { removeUserFromGroup } from '@/app/api/public-cloud/aws-roles/route';
+import { NextRequest, NextResponse } from 'next/server';
+
+export async function DELETE(req: NextRequest, res: NextResponse) {
```

Comment:
Would you try applying the `api handler` for the new endpoints?
I believe it would simplify the codebase:
https://github.com/bcgov/platform-services-registry/blob/main/app/api/example/%5Bcolor%5D/route.ts

Reply:
changes are made, thank you!